README.md

```diff
@@ -8,6 +8,33 @@ Demo: https://www.youtube.com/watch?v=HXdQd65CwkU
 ## Status
 WIP but playable.
 
+## Installation
+
+Prebuilt binary releases are available if you're looking for testing the ruleset or don't possess a developpment environment:
+| [Releases Page](https://github.com/swoolcock/rush/releases/) |
+| -------------|
+
+**NOTE**: For the moment, only desktop platforms (Windows, mac OSX, Linux) are supported.
+
+The ruleset consists of a single DLL file that you'll need to drop in the `rulesets` directory of osu!lazer data directory.
+
+### Installation instructions
+
+1. Navigate to the osu!lazer data directory. You can navigate to it by opening the osu! settings panel and using "open osu! folder" button. Alternatively you can directly navigate to the `rulesets` folder via your OS directory explorer at the following locations:
+    - `%AppData%/osu/rulesets` on Windows
+    - `~/.local/share/osu/rulesets` on Linux / mac OSX 
+
+    **NOTE**: These paths may be incorrect if you have relocated your osu! data directory to another directory.
+
+2. Drag and drop into the `rulesets` directory the ruleset DLL file.
+
+3. Have fun!
+  If lazer was running while installing the ruleset, you may need to restart the game in order for the ruleset to appear in-game.
```

Comment:
```suggestion
  If osu!lazer was running while installing the ruleset, you may need to restart it in order for the ruleset to be acknowledged and loaded for playing.
```

Reply:
In @Game4all 's original sentence, just change `lazer` to `osu!lazer` and remove `in-game`.